src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -307,5 +307,12 @@ trait MarathonConf
     noshort = true,
     hidden = true,
     default = Some(MesosHeartbeatMonitor.DEFAULT_HEARTBEAT_FAILURE_THRESHOLD))
+
+  lazy val drainingTime = opt[Long](
+    "draining_time",
```

Comment:
Actually I was also thinking about this. Probably what it would have sense is to set default to 0, what would mean to directly drop the offers for those hosts that have a maintenance scheduled, no matter when the maintenance starts.